packages/biocaml/biocaml.0.10.1/opam

```diff
@@ -0,0 +1,54 @@
+opam-version: "2.0"
+synopsis: "The OCaml Bioinformatics Library"
+description: """
+Biocaml aims to be a high-performance user-friendly library for
+Bioinformatics. See the [website](http://biocaml.org) for detailed API
+documentation. We welcome contributors and feedback from users. Please
+contact us with any comments and suggestions for features you would
+like added.
+"""
+maintainer: "biocaml@googlegroups.com"
+homepage: "http://biocaml.org"
+bug-reports: "https://github.com/biocaml/biocaml/issues"
+license: "LGPL + linking exception"
+dev-repo: "git+https://github.com/biocaml/biocaml.git"
+authors: [
+  "Sebastien Mondet"
+  "Ashish Agarwal"
+  "Philippe Veber"
+  "Christophe Troestler"
+  "Sergei Lebedev"
+  "David Koppstein"
+  "Alexander Gryzlov"
+  "Francois Berenger"
+]
+
+build: ["dune" "build" "-p" name "-j" jobs]
+
+depends: [
+  "base64"
+  "dune" {build}
```

Comment:
```suggestion
  "dune" {>= "1.0"}
```
Dune should not be tagged as a build dependency (see #14266)

Reply:
Could you return this fix upstream?

Reply:
Thanks for looking at it! 
This is done in https://github.com/biocaml/biocaml/commit/11c6e1a330ed9e923e002e87ac5a922821742837